joomla/joomla-cms
# fix colour => color for the tinymce options

Just a very minimal language fix.
#### How to test
- go to the TinyMCE options
- switch to the advanced tab
- see the option `Colours`
- apply this patch
- see the correct option `Color`

Thanks to @bembelimen
